simp_le.py

```diff
@@ -338,15 +338,14 @@ class ExternalIOPlugin(OpenSSLIOPlugin):
 
     - whenever the script is called with `persisted` as the first
       argument, it should send to STDOUT a subset of three keywords:
-      `key`, `cart`, `chain` (in any order, using any separation
-      character);
+      `key`, `cart`, `chain` (in any order, separated by whitespace);
```

Comment:
I think we agreed to make keywords appear precisely on the first line of STDOUT, and separated by whitespace. Please make it explicit.